tests/src/test/java/io/apicurio/tests/smokeTests/apicurio/AllArtifactTypesIT.java

```diff
@@ -15,116 +15,116 @@
  */
 package io.apicurio.tests.smokeTests.apicurio;
 
-import static io.apicurio.tests.Constants.SMOKE;
-import static org.junit.jupiter.api.Assertions.assertEquals;
-import static org.junit.jupiter.api.Assertions.assertNotNull;
-import static org.junit.jupiter.api.Assertions.fail;
-
-import javax.ws.rs.WebApplicationException;
-
-import org.junit.jupiter.api.Tag;
-import org.junit.jupiter.api.Test;
-
+import io.apicurio.registry.client.RegistryService;
 import io.apicurio.registry.rest.beans.ArtifactMetaData;
 import io.apicurio.registry.rest.beans.Rule;
 import io.apicurio.registry.types.ArtifactType;
 import io.apicurio.registry.types.RuleType;
 import io.apicurio.registry.utils.IoUtil;
+import io.apicurio.registry.utils.tests.RegistryServiceTest;
+import io.apicurio.registry.utils.tests.TestUtils;
 import io.apicurio.tests.BaseIT;
 import io.apicurio.tests.utils.subUtils.ArtifactUtils;
+import org.junit.jupiter.api.Tag;
+
+import static io.apicurio.tests.Constants.SMOKE;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
 
 @Tag(SMOKE)
 class AllArtifactTypesIT extends BaseIT {
     
-    void doTest(String v1Resource, String v2Resource, ArtifactType atype) throws Exception {
-        String artifactId = getClass().getSimpleName() + "-" + atype.name();
+    void doTest(RegistryService apicurioService, String v1Resource, String v2Resource, ArtifactType atype) {
+        String artifactId = TestUtils.generateArtifactId();
```

Comment:
Here too - we're going to be losing potentially important context info.  Perhaps a variant of these `generate` methods that can take a string to use as part of the name?

Reply:
Nah, this doesn't pose any useful info, at least it didn't for me.